axelor-account/src/main/resources/views/Invoice.xml

```diff
@@ -2575,6 +2590,28 @@
     <call class="com.axelor.apps.account.web.InvoiceController" method="setDiscountDummies"/>
   </action-method>
 
+  <action-method name="action-invoice-method-domain-company-tax-number">
+    <call class="com.axelor.apps.account.web.InvoiceController"
+      method="setCompanyTaxNumberDomain"/>
+  </action-method>
+
+  <action-method if-module="axelor-supplychain"
```

Comment:
I don't understand the `if-module="axelor-supplychain"` part, everything is in account module